engine-rest/engine-rest-jakarta/pom.xml

```diff
@@ -0,0 +1,646 @@
+            </goals>
+            <configuration>
+              <outputDirectory>${project.build.directory}/generated-test-sources/jakarta</outputDirectory>
+              <resources>
+                <resource>
+                  <directory>${basedir}/../engine-rest/src/test/java</directory>
+                  <filtering>false</filtering>
+                  <excludes>
+                      <exclude>**/TomcatServerBootstrap.java</exclude>
+                  </excludes>
+                </resource>
+              </resources>
+            </configuration>
+          </execution>
+          <execution>
+            <id>copy-test-sources-resteasy</id>
+            <phase>generate-test-sources</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${project.build.directory}/generated-test-sources/jakarta-resteasy</outputDirectory>
+              <resources>
+                <resource>
+                  <directory>${basedir}/../engine-rest/src/test/java-resteasy</directory>
+                  <filtering>false</filtering>
+                  <excludes>
+                    <exclude>**/ResteasyTomcatServerBootstrap.java</exclude>
+                    <exclude>**/ResteasySpecifics.java</exclude>
+                  </excludes>
+                </resource>
+              </resources>
+            </configuration>
+          </execution>
+          <execution>
+            <id>copy-test-resources</id>
+            <phase>generate-test-resources</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${project.build.directory}/generated-test-resources/jakarta</outputDirectory>
+              <resources>
+                <resource>
+                  <directory>${basedir}/../engine-rest/src/test/resources</directory>
+                  <filtering>true</filtering>
+                </resource>
+              </resources>
+            </configuration>
+          </execution>
+          <execution>
+            <id>copy-test-resources-resteasy</id>
+            <phase>generate-test-resources</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${project.build.directory}/generated-test-resources/jakarta-resteasy</outputDirectory>
+              <resources>
+                <resource>
+                  <directory>${basedir}/../engine-rest/src/test/resources-resteasy</directory>
+                  <filtering>false</filtering>
+                </resource>
+              </resources>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
+        <groupId>org.eclipse.transformer</groupId>
+        <artifactId>transformer-maven-plugin</artifactId>
+        <version>0.5.0</version>
```

Comment:
❓ Can we add the transformer plugin to a higher pom's `pluginManagement` section? We use it at multiple places in our codebase, and some configuration options might be generalized as a baseline (at least, this should be true for the version).